client/pages/BoardPage.jsx

```diff
@@ -30,31 +32,26 @@ function HomePage({user, isLoggedIn, setLogin}) {
   //           }
   //       ]
   //   }
-  // ]
+  // 
     //This is real code do not delete:
     let renderColumns = [];
 
     useEffect(() => {
+      const getBoard = async () => {
+        try {
+          const response = await axios.get(`/boards/${board_id}`);
+          setBoardData(response.data);
+        } catch (e) {
+          console.log('error in useEffect get board: ', e.message);
+        }
+      }
+      getBoard();
+    }, [])
```

Comment:
I think this should work

```suggestion
    useEffect(async () => {
      try {
            const response = await axios.get(`/boards/${board_id}`);
            setBoardData(response.data);
          } catch (e) {
            console.log('error in useEffect get board: ', e.message);
          }
      }, [])
```

Reply:
Perhaps it will, however in my experience with useEffect in the past, React will throw errors if you try using async directly on the anonymous function passed in. Because there is an option to send a cleanup function in to useEffect to mimic componentWillUnmount, React tends to freak out if you do not make a separately defined async function and call it. It gets confused and thinks you're trying to provide a cleanup which is unpure.

Reply:
Okay that's fair and we can just leave it as is